src/test/java/de/tum/in/www1/artemis/FileIntegrationTest.java

```diff
@@ -84,11 +87,29 @@ class FileIntegrationTest extends AbstractSpringIntegrationBambooBitbucketJiraTe
     @Autowired
     private UserRepository userRepository;
 
+    @Autowired
+    private ExamUtilService examUtilService;
+
     @BeforeEach
     void initTestCase() {
         userUtilService.addUsers(TEST_PREFIX, 1, 1, 0, 1);
     }
 
+    @Test
+    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
+    void testUploadExamUserSignature() throws Exception {
+        var course = courseUtilService.addEmptyCourse();
+        var exam = examUtilService.setupExamWithExerciseGroupsExercisesRegisteredStudents(TEST_PREFIX, course, 1);
+        var user = new ExamUserDTO(TEST_PREFIX + "student1", null, null, null, null, null, "", "", true, true, true, true, null);
+        var file = new MockMultipartFile("file", "file.png", "application/json", "some data".getBytes());
+
+        var postResponse = request.postWithMultipartFile("/api/courses/" + course.getId() + "/exams/" + exam.getId() + "/exam-users", user, "examUserDTO", file, ExamUser.class,
```

Comment:
Could you maybe rename the variable `postResponse`? As it is now, I think it is not very clear what it actually is.